src/notation/view/widgets/editstyle.ui

```diff
@@ -11235,7 +11241,7 @@
                  <item row="3" column="4">
                   <widget class="QLabel" name="label_minDistance">
                    <property name="text">
-                    <string>Min. gap between lyrics:</string>
+                    <string>Min. gap between syllables/words:</string>
```

Comment:
This is the only one that we don't have a definitive agreement about internally, so still waiting for that

Reply:
OK, I'll take it out of ths PR and put it into a separate one

Reply:
See #23633